SonarQube.TeamBuild.PreProcessor/WebClientDownloader.cs

```diff
@@ -14,11 +14,11 @@ namespace SonarQube.TeamBuild.PreProcessor
 {
     public class WebClientDownloader : IDownloader
     {
-        private readonly WebClient Client;
+        private readonly WebClient client;
 
-        public WebClientDownloader(WebClient client, string username, string password)
+        public WebClientDownloader(string username, string password)
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) In member 'WebClientDownloader.WebClientDownloader(string, string)', the discrete term 'username' in parameter name 'username' should be expressed as a compound word, 'userName'. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://nemo.sonarqube.org/coding_rules#rule_key=fxcop%3ACompoundWordsShouldBeCasedCorrectly)